src/main/java/com/uniqueAuction/domain/config/SecurityConfig.java

```diff
@@ -0,0 +1,51 @@
+package com.uniqueAuction.domain.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.crypto.argon2.Argon2PasswordEncoder;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.DelegatingPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.crypto.password.Pbkdf2PasswordEncoder;
+import org.springframework.security.crypto.scrypt.SCryptPasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig {
+	@Bean
+	public PasswordEncoder passwordEncoder() {
+		String encodingType = "bcrypt";
+		return createPasswordEncoder(encodingType);
```

Comment:
`passwordEncoder()`에서 하고싶은 작업이 `BCryptPasswordEncoder` 를 반환하는게 맞나요?

그렇다면 단순히 생성후 반환해주어도 문제가 되지 않을것 같아요.
```suggestion
		return new BCryptPasswordEncoder();
```

`createPasswordEncoder`에서도 불필요하게 encoder map을 생성할 필요가 없을것 같습니다.